.golangci.yml

```
@@ -0,0 +1,143 @@
+    - goconst
+    - prealloc
+    - nakedret
+    - nolintlint
+    - whitespace
+
+linters-settings:
+  govet:
+    enable-all: true
+    disable:
+      - fieldalignment
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)